README.md

```diff
@@ -1,10 +1,188 @@
 # kitchen-vcair
 
-This repo is for tracking work on the Test Kitchen driver for vCloud Air.
+A driver to allow Test Kitchen to consume vCloud Air resources to perform testing.
 
-A driver for vCloud Air exists and can be found here:
-https://github.com/vulk/kitchen-vcair
+## Installation
 
-This work is being refactored and will be posted here.  You may follow
-the work in the "refactor" branch of this repository.
+Add this line to your application's Gemfile:
 
+```ruby
+gem 'kitchen-vcair'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install kitchen-vcair
+
+Or even better, install it via ChefDK:
+
+    $ chef gem install kitchen-vcair
+
+## Usage
+
+After installing the gem as described above, edit your .kitchen.yml file to set the driver to 'vcair' and supply your login credentials:
+
+```yaml
+driver:
+  name: vcair
+  vcair_username: user@domain.com
+  vcair_password: MyS33kretPassword
+  vcair_api_host: some-host.vchs.vmware.com
+  vcair_org: M12345678-4321
+```
+
+Additionally, the following parameters are required:
+
+ * **vdc_id** or **vdc_name**: The ID or name of the vDC in which to create your vApp/VM.
+ * **catalog_id** or **catalog_name**: The ID or name of the catalog that contains your image/template.
+ * **image_id** or **image_name**: The ID or name of the image you wish to use to create your VM.
+ * **network_id** or **network_name**: The ID or name of the network to which to attach to your VM.
+
+There are a number of optional parameters you can configure as well:
+
+ * **cpus**: The number of vCPUs to configure for your VM. Default: 1
+ * **memory**: The amount of RAM, in MB, to configure for your VM. Default: 1024
+ * **vcair_api_path**: The URI path for the compute API. This needs to be set when using vCloud Air OnDemand. Default: /api
+ * **vm_password**: The password to set via VM customization for the root/administrator user.
+   * Be sure to set the same password in your `transport` section, too!
+   * NOTE: see the *known issues* section below regarding Windows and passwords.
+
+All of the above settings can be set globally (in the top-level `driver` section), or can be set individually for each platform. For example, you may wish to set your vDC and network globally, but set your catalog and image for each individual platform, and increase the vCPUs/RAM assigned to your windows node:
+
+```yaml
+driver:
+  name: vcair
+  vcair_username: user@domain.com
+  vcair_password: MyS33kretPassword
+  vcair_api_host: some-host.vchs.vmware.com
+  vcair_org: M12345678-4321
+  vdc_name: MyCompany VDC 1
```

Comment:
It depends on how the VDC was created.  We just pass what the user supplies to the API.  If the VDC has spaces in the name, you HAVE to include the spaces here.  Do you think that requires documenting on our side when it's a construct imposed by VMware, not us?